### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by Dunedan** [streetcomplete/StreetComplete#4270] [Feedback]
I just want to thank you for this great app!

I've been using StreetComplete on and off for a few years now, mainly during vacations to improve OSM coverage in places where I haven't been before. What I love about StreetComplete is how frictionless it makes to contribute to OSM: Once you got a few minutes somewhere, flip open the app, solve some quests, done. If that wouldn't be enough, whenever I come back to a new version after a few months, I'm truly astonished how it got even better than it was before. For me StreetComplete is easily one of the best FOSS Android apps!

Thank you for all the work you put into StreetComplete and please keep going!

**Comment by Greenheart** [comment]
I just started using the app and I love it so far! Really inspiring to see the value created by this community already - and the potential of what this could become with more active users! So I'll happily recommend it to my network!

The Pokémon Go devs have to watch out... 😉
